index.html

```diff
@@ -38,8 +41,190 @@ <h1>I'm Mucha <br> Glad to see <br> you!</h1>
                 <li><a href="#"><img src="images/medium.svg" alt="medium"> </a></li>
```

Comment:
- [ ] [OPTIONAL] For portfolio setup, kindly ensure that the social links don't exceed the container boundaries.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/219584765-2c50b9d1-8998-4aef-bba8-817216b83137.png)